src/commands/dev/server_config/mod.rs

```diff
@@ -14,9 +18,11 @@ impl ServerConfig {
     pub fn new(
         host: Option<&str>,
         ip: Option<&str>,
-        port: Option<&str>,
+        port: Option<u16>,
     ) -> Result<Self, failure::Error> {
-        let port = port.unwrap_or("8787");
+        println!("finding open port from ${:?}", port);
+        let port = find_open_port(port, DEFAULT_PORT)?;
+        println!("found open port: {:?}", port);
```

Comment:
if i start a dev process w/using default 8787 port, and then start another one, this prints 

`finding open port from None`
`found open port: 8787`

and then it tries to bind to 8787 and it panics w/address already in use